app/services/idv/flows/doc_auth_flow.rb

```diff
@@ -5,7 +5,8 @@ class DocAuthFlow < Flow::BaseFlow
         welcome: Idv::Steps::WelcomeStep,
         agreement: Idv::Steps::AgreementStep,
         upload: Idv::Steps::UploadStep,
-        send_link: Idv::Steps::SendLinkStep,
+        **(IdentityConfig.store.doc_auth_combined_hybrid_handoff_enabled ?
+          {} : { send_link: Idv::Steps::SendLinkStep }),
```

Comment:
just wanted to flag that adding conditional definitions of constants can be tricky for testing, because the tests could be created with the wrong value. From the description, it sounded like this is temporary, so I'm not too worried

Reply:
Good point. In this case I know at least some of the specs were working because I had the values in the wrong order to start with and it made a test fail.